app/components/efl_qualification_card_component.html.erb

```diff
@@ -0,0 +1,27 @@
+<div id="english-as-a-foreign-language">
+  <h2 class="govuk-heading-m">English as a foreign language</h2>
+  <p class="govuk-body"><%= qualification_status %></p>
+
+  <% if english_proficiency.has_qualification? %>
+    <div class="govuk-grid">
+      <div class="govuk-grid-row app-grid-row--flex">
+        <div class="govuk-grid-column-one-third">
+          <div class="app-card app-card--outline">
+            <h3 class="govuk-heading-s govuk-!-margin-bottom-1"><%= name %></h3>
+            <dl class="app-qualification">
+              <dt class="app-qualification__key govuk-visually-hidden">Awarded</dt>
+              <dd class="app-qualification__value app-qualification__value--caption"><%= award_year %></dd>
+              <dt class="app-qualification__key"><%= grade_title %></dt>
+              <dd class="app-qualification__value"><%= grade %></dd>
+              <dt class="app-qualification__key"><%= reference_number_title %></dt>
+              <dd class="app-qualification__value"><%= unique_reference_number %></dd>
```

Comment:
@malcolmbaig All of this HTML should not appear if there is no reference number, not just the text within in it (affects spacing in card, otherwise). 

```suggestion
              <dt class="app-qualification__key"><%= reference_number_title %></dt>
              <dd class="app-qualification__value"><%= unique_reference_number %></dd>
```